moveos/smt/src/jellyfish_merkle/test_helper.rs

```diff
@@ -46,7 +46,10 @@ pub(crate) fn init_mock_db(
         let (_root_hash, write_batch) = tree
             .insert_all(
                 current_state_root,
-                vec![(key.clone().into_object(), value.clone().into_object())],
+                vec![(
+                    key.clone().into_object().unwrap(),
+                    value.clone().into_object().unwrap(),
```

Comment:
unwarp will cause panic when error occurs

Reply:
Yes, it will be.
But considering this is in test code, it shouldn't cause any Denial Of Service issues.